extensions/2.0/Khronos/KHR_texture_ktx/README.md

```diff
@@ -0,0 +1,293 @@
+        },
+        {
+            "mimeType": "image/ktx2",
+            "bufferView": 2
+        }
+    ]
+}
+```
+
+### Using Without a Fallback
+
+To use KTX v2 image without a fallback, define `KHR_texture_ktx` in both `extensionsUsed` and `extensionsRequired`. The `texture` object will then have its `source` property omitted as shown below.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "extensionsRequired": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+### JSON Schema
+
+[texture.KHR_texture_ktx.schema.json](schema/texture.KHR_texture_ktx.schema.json)
+
+## Color Texture Sources
+
+Texture data provided by this extension is always encoded using linear transfer function. When this extension is used with material texture slots that expect sRGB-encoded values, only the fallback 8-bit texture (when present) uses sRGB encoding.  
+
+>Implementation Note: For the core glTF 2.0 specification, these texture slots include emissiveTexture and baseColorTexture; for material extensions, it's usually texture slots named *ColorTexture.  
+
+## KTX v2 Images
+
+For the purposes of this extension, the following texture types and formats are defined:
+
+- **RGBA:** A texture that uses all four channels.  
+
+VK_FORMAT_R16G16B16A16_SFLOAT  
+VK_FORMAT_R16G16B16A16_UNORM  
+VK_FORMAT_R32G32B32A32_SFLOAT  
+
+
+- **RGB:** A texture that uses red, green, and blue channels. Alpha channel is unused and not sampled at runtime.  
+
+VK_FORMAT_E5B9G9R9_UFLOAT_PACK32  
+VK_FORMAT_B10G11R11_UFLOAT_PACK32  
+VK_FORMAT_R16G16B16_SFLOAT  
+VK_FORMAT_R16G16B16_UNORM  
+VK_FORMAT_R32G32B32_SFLOAT  
+
+- **RG_** A texture that uses, red and green channels. All other channels are unused and their values are not sampled at runtime.  
+
+VK_FORMAT_R16G16_SFLOAT  
+VK_FORMAT_R16G16_UNORM  
+VK_FORMAT_R32G32_SFLOAT  
+
+
+- **Red:** A texture that uses only red channel. All other channels are unused and their values are not sampled at runtime.  
+
+VK_FORMAT_R16_SFLOAT  
+VK_FORMAT_R16_UNORM  
+VK_FORMAT_R32_SFLOAT  
+
+**Implementation Notes**  
+This section is non-normative
+
+This extension defines a number of formats that are supported for images used as texture sources.  
+It does not define how these formats are stored internally, it is up to implementations to use a suitable texture format.  
+The source texture shall be used in shader (BRDF) calculations in a way that retains the increased range.  
+However, this extension does not define the way pixel data is written to framebuffer or how swapchain is presented using the increased range.  
+
+If ktx image 
+
+## KTX header fields
+
+`supercompressionScheme` MUST be 3 (ZLIB).  
```

Comment:
Why this restriction? Zstandard is faster and more efficient.

Reply:
Just because I'm implementing using ZLIB - I'll add Zstandard as well.